_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+
+Just like Hardware Outputs and Virtual Inputs, there are different features added in Banana and Potato than what comes with Base Voicemeeter. Base Voicemeeter gets the gain slider, a mute, mono selector ( triggers mono channeling where the same signal is sent equally through both stereo outputs. It helps some mics that for some reason only pull in through one side in some cases and it helps to balance that one sided input to both ears on output.) Banana gains a compressor and noise gate over Base Voicemeeter. Potato gains Special Effects: Reverb, Delay, and separate Sends for each. I will give an image per program to show what they look like:
+
+Base Voicemeeter Hardware Inputs:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Voicemeeter Banana Hardware Inputs:
+{
+
+
+
+  Need picture here
+
+
+
+}
+
+Voicemeeter Potato Hardware Inputs:
+{
+
+
+
+  Need Picture Here:
+
+
+
+}
+
+Alright, that explains the whole of voicemeter right? Go have fun with it! I just, I jest. That was mostly just trying to help you understand each part before running through a setup. When is setup? Setup is now! and it is the fun part! First off, we are going to do something different than I have seen a lot of guides do. We are going to skip around to setting up our outputs first. This way we can leave our current setup alone and you can listen to music or do other things, maybe even come say hello to use in the Library discord? [liiiink the discord here] We would be glad to ahve you and discuss things. Semi-guilt-feeling discord link aside, lets get back into setting up Voicemeeter!
+
+This might seem weird but,.. Lets skip regular Voicemeeter and go straight to that Banana! Yeah! Ba na na... You thought I was going to sing there? (I totally was just now), Anyways, my reason for skipping Base Voicemeeter is actually because I honestly feel like it lacks a lot of functionality that just warrants skipping it, especially since Banana is the same price of free and offers way more features and functionality. If you haven't already, this is where you would want to install Voicemeeter Banana and the extra Virtual Audio Cable(s). If you installed Voicemeeter Potato, it also installs Voicemeeter Banana. Got everything installed? Cool! Open up Voicemeeter Banana please and this is what you should see:
+
+Voicemeeter Banana Full Program Image Pre-Setup:
+{
+
+
+
+  Need Picture Here:
+
+
+
+}
+
+Something that typically gets skipped in other Voicemeeter coverage that I've seen is the ability to save settings. At any point you can save your current settings of Voicemeeter by going to Menu -> Save Settings. This saves a settings file on your computer you can then load up from just a file even on other devices should you want to transfer your settings to another device. If at any point you feel lost and want to restart the setup, you can go to Menu -> Reset Settings ( Re-Initialization) and this will make Voicemeeter appear as if you had just installed it, blank with nothing set up.
```

Comment:
Saving settings is a good call. Is there a way to import settings files to get them back to a good state without needing to start over from scratch?

Reply:
Maybe I should cover loading the previous settings as well as saving? My assumption was, once someone saw the save / load / reset, then learned to save settings, they would automatically know how to load but I could take a couple more snapshots to add more depth to that walk through if you think I should?

Reply:
added wording about how to load the saved files